rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -184,8 +184,91 @@ public Sw360Module() {
+                "leadArchitect",
+                "moderators",
+                "contributors",
+                "visbility",
+                "clearingTeam",
+                "phaseOutSince",
+                "homepage",
+                "wiki",
+                "documentState",
+                "releaseClearingStateSummary",
+                "permissions",
+                "attachmentsIterator",
+                "moderatorsIterator",
+                "contributorsIterator",
+                "releaseIdsIterator",
+                "setId",
+                "setRevision",
+                "setType",
+                "setName",
+                "setDescription",
+                "setVersion",
+                "setExternalIds",
+                "setAttachments",
+                "setCreatedOn",
+                "setState",
+                "setProjectType",
+                "setTag",
+                "setCreatedBy",
+                "setModerators",
+                "setVisbility",
+                "setHomepage",
+                "externalIdsSize",
+                "attachmentsSize",
+                "setBusinessUnit",
+                "setProjectResponsible",
+                "setLeadArchitect",
+                "moderatorsSize",
+                "contributorsSize",
+                "setContributors",
+                "linkedProjectsSize",
+                "setLinkedProjects",
+                "releaseIdToUsageSize",
+                "setReleaseIdToUsage",
+                "setClearingTeam",
+                "setPreevaluationDeadline",
+                "setSystemTestStart",
+                "setClearingSummary",
+                "setObligationsText",
+                "setSpecialRisksOSS",
+                "setGeneralRisks3rdParty",
+                "setSpecialRisks3rdParty",
+                "setDeliveryChannels",
+                "setRemarksAdditionalRequirements",
+                "setSystemTestEnd",
+                "setDeliveryStart",
+                "setPhaseOutSince",
+                "setDocumentState",
+                "releaseIdsSize",
+                "setReleaseClearingStateSummary",
+                "permissionsSize",
+                "setWiki",
+                "setReleaseIds",
+                "setPermissions",
+                "setClearingState",
+                "setTodos",
+                "todosSize",
+                "securityResponsiblesSize",
+                "securityResponsiblesIterator",
+                "setSecurityResponsibles",
+                "setOwnerGroup",
+                "setOwnerCountry",
+                "rolesSize",
+                "setRoles",
+                "setOwnerAccountingUnit",
+                "setLicenseInfoHeaderText",
+                "setProjectOwner",
+                "setEnableSvm",
+                "setEnableVulnerabilitiesDisplay",
+                "enableSvm",
+                "enableVulnerabilitiesDisplay"
 	})
 	static abstract class EmbeddedProjectMixin extends ProjectMixin {
```

Comment:
These should not be necessary, since `EmbeddedProjectMixin` extends `ProjectMixin` and thus also derives the Ignore list from `ProjectMixin`. See discussion in https://github.com/eclipse/sw360/pull/531

Reply:
Unfortunately, It's not behaving this way right now. There might be some deployment issue while testing the PR #531 in local environment previously. But while testing it in higher environment, we found that it's not behaving correctly, as it should.

Reply:
But then there has to be a way to reduce duplicated code or to enforce, that the list for `EmbeddedProjectMixin` is larger then `ProjectMixin`

Reply:
I have reduced the duplicated code. Kindly re-review.